src/UserService.Validation/User/EditUserRequestValidator.cs

```diff
@@ -91,8 +91,8 @@ private void HandleInternalPropertyValidation(Operation<EditUserRequest> request
             AddСorrectOperations(nameof(EditUserRequest.Rate), new List<OperationType> { OperationType.Replace });
             AddСorrectOperations(nameof(EditUserRequest.Gender), new List<OperationType> { OperationType.Replace });
             AddСorrectOperations(nameof(EditUserRequest.City), new List<OperationType> { OperationType.Replace, OperationType.Add, OperationType.Remove });
-            AddСorrectOperations(nameof(EditUserRequest.DateOfBirth), new List<OperationType> { OperationType.Replace, OperationType.Add, OperationType.Remove });
-            AddСorrectOperations(nameof(EditUserRequest.StartWorkingAt), new List<OperationType> { OperationType.Replace, OperationType.Add, OperationType.Remove });
+            AddСorrectOperations(nameof(EditUserRequest.DateOfBirth), new List<OperationType> { OperationType.Replace, OperationType.Add });
```

Comment:
I think you can remove "add" operation

Reply:
why do you remove Remove

Reply:
@MariyaSin why? Let it be
@BelehovEgor I am agree

Reply:
we discussed it yesterday